sections/main-footer.liquid

```diff
@@ -14,13 +14,23 @@
       <div class='footer-logo column'>
         <img loading='lazy' src='{{ section.settings.footer_logo.src }}'>
       </div>
+    {% else %}
+      <div class='footer-logo column'>
+        <img loading='lazy' src='{{ 'default_product.jpeg' | asset_url }}'>
+      </div>
```

Comment:
no need to display the default image as i said you we want to have the logo of the theme hire like the design